cypress/e2e/layers.cy.ts

```diff
@@ -263,165 +315,183 @@ describe("layers", () => {
         it.skip("parse error", () => {
           var bgId = createBackground();
 
-          driver.click(driver.getDataAttribute("layer-list-item:background:"+bgId));
+          when.click("layer-list-item:background:" + bgId);
 
           var errorSelector = ".CodeMirror-lint-marker-error";
-          driver.doesNotExists(errorSelector);
+          should.notExist(errorSelector);
 
-          driver.click(".CodeMirror");
-          driver.typeKeys("\uE013\uE013\uE013\uE013\uE013\uE013\uE013\uE013\uE013\uE013\uE013\uE013 {");
-          driver.isExists(errorSelector);
+          when.click(".CodeMirror");
+          when.typeKeys(
+            "\uE013\uE013\uE013\uE013\uE013\uE013\uE013\uE013\uE013\uE013\uE013\uE013 {"
+          );
+          should.isExists(errorSelector);
 
-          driver.click(driver.getDataAttribute("layer-editor.layer-id"));
+          when.click("layer-editor.layer-id");
```

Comment:
Not sure why a click at the end of the test is needed, maybe worth trying to remove it and see if the tests continue to pass, otherwise keep it.